st3/lsp_utils/node_runtime.py

```diff
@@ -26,10 +26,10 @@
 NODE_RUNTIME_VERSION = '18.18.1'
```

Comment:
Do you plan to change this value also to match the electron node version?

Reply:
For sure :)
thanks